app/views/home/_banner.html.haml

```diff
@@ -0,0 +1,25 @@
+.container.p-home__banner
+  .row
+    .col-sm-12
+      %button.home__banner_close.glyphicon.glyphicon-remove-circle
+      = image_tag 'lilo-blue.png', alt: 'Lilo', class: 'home__banner_logo'
+      %h3.p-home__heading Chères utilisatrices et chers utilisateurs,
+      %p
+        Covoiturage-libre.fr est un site associatif à but non lucratif dont le service de mise en relation entres covoitureurs est gratuit. Cependant le développement a un coût. Nous sommes financés par les dons des usagers qui soutiennent notre démarche.
+      %p
+        <strong>Grâce au moteur de recherche Lilo, vous pouvez nous financer sans bourse délier</strong>, simplement en faisant vos recherches web avec Lilo, et en nous donnant les "gouttes d'eau" que vous gagnez à chaque recherche web. Merci pour votre soutien&nbsp;!
+  .row
+    .col-sm-12
+      = link_to 'Découvrir le moteur de recherche Lilo', 'https://www.lilo.org/?utm_source=covoiturage-libre-fr', target: '_blank', rel: 'noopener', class: 'btn btn-default'
+      = link_to 'Déjà sur Lilo ? Donnez-nous vos gouttes !', 'https://www.lilo.org/?p=9680&utm_source=covoiturage-libre-fr', target: '_blank', rel: 'noopener', class: 'btn btn-default'
+
+- content_for :javascript_footer do
+  :javascript
+    // Close button for banner
+    $(".home__banner_close").on("click", function() {
```

Comment:
You can put selectors like `$(".home__banner_close")` in variables to avoid research.

Reply:
Optimize ! Thanks for the review !